### ISSUE TITLE
type scatter resets expected default value of dataGrouping.groupPixelWidth

### ISSUE THREAD

**Issue by Sicaine** [highcharts/highcharts#2915]
Hi,

If you have 2 series and one of them is a scatter, the expected default value for dataGrouping.groupPixelWidth is set to 0 instead of 2 as the documentation states:

```
groupPixelWidth: Number
The approximate pixel width of each group. If for example a series with 30 points is displayed over a 600 pixel wide plot area, no grouping is performed. If however the series contains so many points that the spacing is less than the groupPixelWidth, Highcharts will try to group it into appropriate groups so that each is more or less two pixels wide. If multiple series with different group pixel widths are drawn on the same x axis, all series will take the greatest width. For example, line series have 2px default group width, while column series have 10px. If combined, both the line and the column will have 10px by default. Defaults to 2. Defaults to 2.
```

In Highstock.src.js:18787 is a 'merging' behaviour which fails. If one series has a NaN like scatter, instead of returning groupPixelWidth = 2 an NaN will be set which then leads to 0 in 18814.

**Comment by NathanWolf** [comment]
I also ran into this issue, here is a reproduction case:

http://jsfiddle.net/NathanWolf/qz16corL/1/

I think this could easily be resolved around line 20197 (of HighStocks src 2.1.8), by changing "if (dgOptions) {" to "if (dgOptions && dgOptions.groupPixelWidth) {"

**Comment by NathanWolf** [comment]
It seems you can work around this by explicitly specifying "groupPixelWidth: 2" in the series plotOptions of the chart configuration. This will apply to the scatter series and make the Axis.getGroupPixelWidth method avoid the NaN issue.

**Comment by KacperMadej** [comment]
This issues is resolved through PR #9765
Demo: http://jsfiddle.net/BlackLabel/kqb6s05L/
Closing as resolved.
